### PR TITLE
WIP feat: add proven ciphertext

### PR DESCRIPTION
**Pull request by immortal-tofu**

### CODE REVIEW

File: src/sdk/encrypt.ts [hunk 1]
```diff
@@ -20,15 +21,17 @@ export type ZKInput = {
...
   add128: (value: number | bigint) => ZKInput;
+  add256: (value: number | bigint) => ZKInput;
+  addBytes64: (value: Uint8Array) => ZKInput;
+  addBytes128: (value: Uint8Array) => ZKInput;
+  addBytes256: (value: Uint8Array) => ZKInput;
   addAddress: (value: string) => ZKInput;
-  getValues: () => bigint[];
   getBits: () => number[];
-  resetValues: () => ZKInput;
   encrypt: () => {
```
Comment (jatZama) [inline src/sdk/encrypt.ts:30]: can you make this function async please? this is needed to get parity with mocked mode, and I believe it makes more sense anyways wdyt? @immortal-tofu

Comment (immortal-tofu) [reply src/sdk/encrypt.ts:30]: Yes it will be the case when I'll add fetch() to the gateway, don't worry

**Comment by amaury1093** [comment]
Hey @immortal-tofu! Could you add some notes here why you closed the PR? I was actually following and looking forward to test this.

**Comment by immortal-tofu** [comment]
@amaury1093 Here https://github.com/zama-ai/fhevmjs/pull/95

**Pull request closed without merging**